src/diffusers/models/unet_2d_condition.py

```diff
@@ -127,20 +127,48 @@ def __init__(
         num_class_embeds: Optional[int] = None,
         upcast_attention: bool = False,
         resnet_time_scale_shift: str = "default",
+        time_embedding_type: str = "positional",  # fourier, positional
+        timestep_act_2: Optional[str] = None,
```

Comment:
Can we adapt the corresponding doc string here correctly?

Reply:
✅